core/src/main/conf/conf.d/logging.conf

```diff
@@ -0,0 +1,5 @@
+[logging]
```

Comment:
I'm not sure what's the appropriate default values here, correct it if needed pls.